projects/packages/videopress/src/class-initializer.php

```diff
@@ -193,14 +198,36 @@ public static function register_videopress_video_block() {
 		// Pick the block name straight from the block metadata .json file.
 		$videopress_video_block_name = $videopress_video_metadata->name;
 
-		// Register and enqueue scripts used by the VideoPress block.
-		Block_Editor_Extensions::init( $videopress_video_metadata );
+		// Register and enqueue scripts used by the VideoPress video block.
+		Block_Editor_Extensions::init( self::JETPACK_VIDEOPRESS_VIDEO_HANDLER );
 
 		// Do not register if the block is already registered.
 		if ( \WP_Block_Type_Registry::get_instance()->is_registered( $videopress_video_block_name ) ) {
 			return;
 		}
 
+		// Register script used by the VideoPress video block in the editor.
+		Assets::register_script(
+			self::JETPACK_VIDEOPRESS_VIDEO_HANDLER,
+			'../build/block-editor/blocks/video/index.js',
+			__FILE__,
+			array(
+				'in_footer'  => false,
+				'textdomain' => 'jetpack-videopress-pkg',
+			)
+		);
+
 		register_block_type( $videopress_video_metadata_file );
```

Comment:
Probably makes no difference, but just in case I'd move this to after the second `Assets::register_script()` call.

Reply:
https://github.com/Automattic/jetpack/pull/28965/commits/584448b80587b90c0adfa682bc6d81cb8b64a6f6